HaulageApplication/Haulage/DatabaseExecutionServices/UserExecutionService.cs

```diff
@@ -0,0 +1,71 @@
+﻿using SQLite;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Haulage.BaseClasses;
+using Haulage.BaseClasses.Accounting;
+
+
+namespace Haulage.DatabaseExecutionServices
+{
+    public static class UserExecutionService
+    {
+        public static List<Driver> GetDrivers(SQLiteConnection? connection = null)
+        {
+            var drivers = new List<Driver>();
+            var sql = "SELECT [UserId]" +
```

Comment:
the sql is currently getting all users and not all drivers the roleID property on roleId should be an enum. This is a priority as this would be required for any progress to be made on customer and adminstrator

Reply:
Added Where clause to SQL to display only drivers where Role = 'Driver'

Reply:
enum has replaced string for role